core/templates/services/image-upload-helper.service.ts

```diff
@@ -41,27 +41,28 @@ export class ImageUploadHelperService {
   }
 
   // Image file returned will be null when blob is not of type
-  // 'image' or blob size is zero.
-  convertImageDataToImageFile(dataURI: string): Blob | null {
+  // 'image', blob size is zero or dataURI does not exists.
+  convertImageDataToImageFile(dataURI: string | null): Blob | null {
```

Comment:
why can dataURI be null?

Reply:
Storage.getItem can return a null value if filename doesn't exist in localStorage, updated the comment
![image](https://user-images.githubusercontent.com/60519303/128558074-91ec063e-c7ac-4d10-8e97-f9628b4f16e2.png)

Reply:
Reverted, using a different approach now